app-tasks/dags/export/find_exports.py

```diff
@@ -0,0 +1,117 @@
+import os
+from collections import namedtuple
+import datetime
+import logging
+import json
+import boto3
+from time import time
+
+from airflow.bin.cli import trigger_dag
+from airflow.models import DAG
+from airflow.operators.python_operator import PythonOperator
+
+from rf.utils.io import get_session
+from rf.utils.exception_reporting import wrap_rollbar
+
+rf_logger = logging.getLogger('rf')
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+rf_logger.addHandler(ch)
+
+logger = logging.getLogger(__name__)
+
+default_args = {
+    'owner': 'raster-foundry',
+    'start_date': datetime.datetime(2017, 1, 1)
+}
+
+base_params = {'exportStatus': 'NOTEXPORTED'}
+
+DagArgs = namedtuple('DagArgs', 'dag_id, conf, run_id')
+
+
+HOST = os.getenv('RF_HOST')
+API_PATH = '/api/exports/'
+#############
+# Constants #
+#############
+
+FIND_TASK_ID = 'find_exports'
+#KICKOFF_TASK_ID = 'kickoff_export_checks'
+UPDATE_DAG_ID = 'do_export'
+
+# UTILS
+def update_status(export, status):
+    url = '{HOST}{API_PATH}{id}/'.format(HOST=HOST, API_PATH=API_PATH, id=export['id'])
+    session = get_session()
+    response = session.get(url)
+    obj = response.json()
+    obj['exportStatus'] = status
+    json_d = json.dumps(obj)
+    update_response = session.put(url, headers={'Content-Type':'application/json'}, data=json_d)
+    try:
+        update_response.raise_for_status()
+    except:
+        logger.exception('Unable to update: %s', update_response.text)
+        raise
+    return update_response
+
+#################################
+# Callables for PythonOperators #
+#################################
+@wrap_rollbar
+def get_exports():
```

Comment:
The possible downside with replacing this, specifically, with an implementation in the JAR is that airflow would then show only a single "uber-task" that was actually managing multiple exports, instead of a task for each export.

That isn't to say that there's a tangible problem per se, as reporting and/or error handling could readily be managed within the `rf-batch` implementation, but rather just that it may be worth considering the visibility from airflow.